feature-libs/quote/components/details/comment/quote-details-comment.component.spec.ts

```diff
@@ -127,6 +127,31 @@ describe('QuoteDetailsCommentComponent', () => {
     expect(component).toBeTruthy();
   });
 
+  describe('Ghost animation', () => {
+    it('should render view for ghost animation', () => {
+      component.quoteDetails$ = of(null);
```

Comment:
strict mode issue - null is not valid here

Reply:
done